core/exceptions/InvalidEntityException.php

```diff
@@ -37,7 +40,7 @@ public function __construct($actual, $expected, $message = '', $code = 0, Except
                     ? get_class($actual)
                     : gettype($actual),
                 $expected,
-                var_export($actual, true)
+                $object
```

Comment:
Wouldn't `var_export` work more often than hoping the object can be turned into a string? This might have warning sometimes right?

Reply:
`$object` is the output buffer contents after var dumping the object...
this code is just before the message code above:

```php
ob_start();
var_dump($actual);
$object = ob_get_clean();
```

Reply:
oh I see, ya I missed that. kk!